tinyosc.h

```diff
@@ -18,6 +18,7 @@
 #define _TINY_OSC_
 
 #include <stdint.h>
+#include <sys/select.h>
```

Comment:
This PR is a nobrainer to merge, but what is this include doing here? Should it be in `main.c`?